packages/shadcn/src/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,71 @@
+import { FormContextType, IconButtonProps, RJSFSchema, StrictRJSFSchema, TranslatableString } from '@rjsf/utils';
+import { Button, ButtonProps } from '../components/ui/button';
+import { ChevronDown, ChevronUp, Copy, Trash2 } from 'lucide-react';
```

Comment:
Keep the absolute imports before the relative ones
```suggestion
import { ChevronDown, ChevronUp, Copy, Trash2 } from 'lucide-react';
import { Button, ButtonProps } from '../components/ui/button';
```